docs/development/development.md

```diff
@@ -0,0 +1,283 @@
+# Developer Guide
+This document will walk you through how to set up a local development environment, build Akri component containers, and test Akri using your newly built containers.
+
+The document includes [naming guidelines](#naming-guidelines) to help as you extend Akri.
+
+## Required Tools
+To develop, you'll need:
+
+- A Linux environment whether on amd64 or arm64v8
+- Rust - version 1.51.0 which the build system uses can be installed using: 
```

Comment:
should rust be version 1.54.0?

Reply:
i was going to put in a new PR to update to 1.54 once https://github.com/deislabs/akri/pull/361 is merged